dataplane/dplanerc/interface.go

```diff
@@ -909,7 +986,7 @@ func (ni *Reconciler) handleAddrUpdate(ctx context.Context, au *netlink.AddrUpda
 		_, err := ni.routeClient.CreateRouteEntry(ctx, &saipb.CreateRouteEntryRequest{
 			Entry: &saipb.RouteEntry{
 				SwitchId:    ni.switchID,
-				VrId:        0,
+				VrId:        ni.niDetail[data.networkInstance].vrOID,
```

Comment:
What is ni.niDetail[data.networkInstance] is nil?

Reply:
added check